src/main/java/com/github/cross_language_cpp/djinni_intellij_plugin/Djinni.bnf

```diff
@@ -91,6 +94,20 @@ enumValue ::= identifier {
   methods=[getName setName getNameIdentifier]
 }
 
+// flags.
+
+private flagsDescription ::= flagsTypeVariant LEFT_BLOCK_BRACE flagsMember* RIGHT_BLOCK_BRACE
```

Comment:
This is where the magic happens.
I basically copied the `enum`-grammar, renamed everything to `flags` and extended it with the `all` and `none` keywords.

Reply:
long time/term goal (note to myself) : 
it would be nice to have only on parser through the whole project, because now we have 2
the regex based one in the generator, and that one used here

Reply:
I agree, in theory it would be super cool to use the same AST in both the generator and the intellij plugin. I doubt that in near future it's worth the effort doing research on this, as long as the plugin "just works ™️"...